README.md

```diff
@@ -50,10 +50,22 @@ To build it, run:
 
     mvn -DskipTests clean package
 
+Build against Spark 3.2.x
```

Comment:
How about add description about default build is for spark 3.1.2, for other version of Spark3, profile should be added?

Reply:
Yes, we should indeed mention the default on the docs.